core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -428,7 +428,8 @@ object AdminClient {
     val retryBackoffMs = config.getLong(CommonClientConfigs.RETRY_BACKOFF_MS_CONFIG)
 
     val brokerUrls = config.getList(CommonClientConfigs.BOOTSTRAP_SERVERS_CONFIG)
-    val brokerAddresses = ClientUtils.parseAndValidateAddresses(brokerUrls)
+    val clientDnsLookup = config.getString(CommonClientConfigs.CLIENT_DNS_LOOKUP)
```

Comment:
`CLIENT_DNS_LOOKUP` => `CLIENT_DNS_LOOKUP_CONFIG` (this is causing the build failure)